packages/components/vite.config.js

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import typescript from '@rollup/plugin-typescript';
```

Comment:
Can we share this vite config between packages since it is non-trivial?

Reply:
Yes, we can share it the same way we share Jest configuration by putting it in a shared package.

Reply:
Done in https://github.com/DataBiosphere/terra-ui/pull/4116/commits/1a1e356ca5ea12672851bb21b17f1b3ae0df3f48.